solr/conf/solrconfig.xml

```diff
@@ -70,9 +70,9 @@
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
   <query>
     <maxBooleanClauses>1024</maxBooleanClauses>
-    <filterCache class="solr.FastLRUCache" size="512" initialSize="512" autowarmCount="0"/>
-    <queryResultCache class="solr.LRUCache" size="512" initialSize="512" autowarmCount="0"/>
-    <documentCache class="solr.LRUCache" size="512" initialSize="512" autowarmCount="0"/>
+    <filterCache class="solr.CaffeineCache" size="512" initialSize="512" autowarmCount="0" async="true"/>
+    <queryResultCache class="solr.CaffeineCache" size="512" initialSize="512" autowarmCount="0" async="true"/>
+    <documentCache class="solr.CaffeineCache" size="512" initialSize="512" autowarmCount="0" async="true"/>
```

Comment:
fwiw I think `async="true"` [is the default](https://solr.apache.org/guide/8_11/query-settings-in-solrconfig.html#cache-implementations):
> `CaffeineCache` supports the `async` attribute, which determines whether the cache stores direct results (`async=false`, disabled) or whether it will store indirect references to the computation (`async=true`, enabled by default).